sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -749,7 +749,11 @@ export class ManagementClient extends LinkEntity {
    * @param options Options that can be set while sending the request.
    * @returns Promise<Date> New lock token expiry date and time in UTC format.
    */
-  async renewSessionLock(sessionId: string, options?: SendRequestOptions): Promise<Date> {
+  async renewSessionLock(
+    sessionId: string,
+    options?: SendRequestOptions,
+    receiverName?: string
```

Comment:
As discussed in the other PR, let's swap the order of `receiverName` and `options`.
Also, lets rename this parameter to `associatedLinkName`.
Given that we intend to do this to some of the other operations as well, `associatedLinkName` is better suited for the purpose it gets used for on the service side

Reply:
updated